taas/config.yml

```diff
@@ -54,3 +54,88 @@ sources:
                 acronym: ACRONYM
                 group_type: Group Type
                 homepage: Homepage
+
+    countries:
+        beta-v1:
+            key: 1NjSI2LaS3SqbgYc0HdD8oIb7lofGtiHgoKKATCpwVdY
+            gid: 1528390745
+            mapping:
+                id: ID
+
+                iso3:
+                    field: ISO 3166-3:2013 Alpha 3-Codes
+                    optional: True
+
+                iso2:
+                    field: ISO 3166-2:2013 Alpha 2-Codes
+                    optional: True
+
+                # Unofficial but generally used fields. These
+                # default to the ISO code where it exists.
+                x-alpha-2:
+                    field:
+                        - x Alpha2 codes
+                        - ISO 3166-2:2013 Alpha 2-Codes
+                    optional: True
+
+                x-alpha-3:
+                    field:
+                        - x Alpha3 codes
+                        - ISO 3166-3:2013 Alpha 3-Codes
+                    optional: True
+
+                # The old HRinfo ID, in case people want to match.
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+
+                # Ditto ReliefWeb
```

Comment:
Replace "Ditto" with the full explanation: `The old ReliefWeb ID, in case people want to match.`